cmd/crc/cmd/delete.go

```diff
@@ -51,13 +52,18 @@ func deleteMachine(client machine.Client, clearCache bool, cacheDir string, inte
 
 	yes := input.PromptUserForYesOrNo("Do you want to delete the OpenShift cluster", force)
 	if yes {
-		return true, client.Delete()
+		err := client.Delete()
+		// Delete the log file
```

Comment:
For what it's worth, since we call a method with a good descriptive name `RemoveLogFile`, I don't think this comment is needed.